.github/workflows/cypress.yml

```
@@ -0,0 +1,36 @@
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install
+      - name: Run E2E Tests for H5
+        env:
+          CI: true
+        run: pnpm cypress install --force && pnpm e2e:run:h5
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)